chat/message/src/Message/Message.tsx

```diff
@@ -148,3 +160,39 @@ export const Message = forwardRef(
 Message.displayName = 'Message';
 
 export default Message;
+
+function VerifiedAnswerBanner({
+  verifier,
+  verifiedAt,
+  learnMoreUrl,
+}: NonNullable<MessageProps['verified']>) {
```

Comment:
Could we just say these props are of type `VerificationInfo`? Do we expect these to potentially be different in the future?

Reply:
Ah yeah good catch! I originally had `VerificationInfo` defined inline and meant to port it over here.